src/ScottPlot4/ScottPlot/ScottPlot.csproj

```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <OutputType>Library</OutputType>
-        <TargetFrameworks>net462;netstandard2.0;net6.0</TargetFrameworks>
+        <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
I don't think we should change the target platform here: Although this builds when targeting .NET standard, all the build targets are intentional because math operations are optimized in a version-specific way (https://github.com/ScottPlot/ScottPlot/pull/1927)